packages/xo-server-audit/src/index.js

```diff
@@ -186,6 +193,33 @@ class AuditXoPlugin {
     return records
   }
 
+  _exportRecords() {
+    return this._xo
+      .registerHttpRequest(
+        (req, res) => {
+          res.writeHead(200, {
+            'content-disposition': 'attachment',
+            'content-type': 'application/json',
+          })
+          return fromCallback(
+            pipeline,
+            this._getRecordsStream(),
+            createGzip(),
+            res
+          )
```

Comment:
Can you explain this to me and why you use `createGzip`? Please ^_^

Reply:
`createGzip` is used to compress a readable stream with `gzip` compression. Please see this [example](https://nodejs.org/docs/latest-v13.x/api/zlib.html#zlib_zlib) to understand the usage of this `zlib` method. I use it to optimize the download of the records (less download time, less bandwidth ...).